tests/trading/test_orders.py

```diff
@@ -23,207 +23,215 @@
 WORSE_ORDER_ID = 6
 GAS_PRICE = 7
 
+ZEROED_ORDER_ID = '\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
+
 def test_walkOrderList_bids(contractsFixture):
     market = contractsFixture.binaryMarket
-    orders = contractsFixture.contracts['orders']
-    ordersFetcher = contractsFixture.contracts['ordersFetcher']
+    orders = contractsFixture.contracts['Orders']
+    ordersFetcher = contractsFixture.contracts['OrdersFetcher']
     outcomeID = 1
     order = {
         "orderID": 5,
-        "type": 1,
+        "type": BID,
         "fxpAmount": fix('1'),
         "fxpPrice": fix('0.6'),
         "sender": tester.a0,
         "outcome": outcomeID,
         "fxpMoneyEscrowed": fix('0.6'),
         "fxpSharesEscrowed": 0,
-        "betterOrderID": 0,
-        "worseOrderID": 0,
+        "betterOrderID": ZEROED_ORDER_ID,
+        "worseOrderID": ZEROED_ORDER_ID,
         "tradeGroupID": 0
     }
-    assert(orders.saveOrder(order["orderID"], order["type"], market.address, order["fxpAmount"], order["fxpPrice"], order["sender"], order["outcome"], order["fxpMoneyEscrowed"], order["fxpSharesEscrowed"], order["betterOrderID"], order["worseOrderID"], order["tradeGroupID"], 1) == 1), "Save order"
-    bestOrderID = int(orders.getBestOrderID(1, market.address, outcomeID), 16)
-    worstOrderID = int(orders.getWorstOrderID(1, market.address, outcomeID), 16)
-    assert(bestOrderID == 5)
-    assert(worstOrderID == 5)
+    hashedOrderID5 = orders.saveOrder(ZEROED_ORDER_ID, order["type"], market.address, order["fxpAmount"], order["fxpPrice"], order["sender"], order["outcome"], order["fxpMoneyEscrowed"], order["fxpSharesEscrowed"], order["betterOrderID"], order["worseOrderID"], order["tradeGroupID"])
```

Comment:
Naming: Variable should just be called `orderId`.  As mentioned previously, the fact that it is a hash is an implementation detail.  Also, it is unclear why this has `5`  on the end?

Reply:
Same comment for other tests in this file.

Reply:
I added the `5` on the end to differentiate it from some of the other orderIds in the file.  Previously, we were just using integers, so lines like these:

`    assert(bestOrderID == 5)
    assert(worstOrderID == 6)`

had to be changed to:

`    assert(bestOrderID == hashedOrderID5)
    assert(worstOrderID == hashedOrderID6)`

I'll change the `hashedOrderID` prefix to `orderId`.